osu.Game/Graphics/Containers/ShakeContainer.cs

```diff
@@ -26,6 +27,11 @@ public class ShakeContainer : Container
         /// </summary>
         public float ShakeMagnitude = 8;
 
+        /// <summary>
+        /// Fired when <see cref="Shake"/> finishes
+        /// </summary>
+        public event Action OnShakeFinish;
```

Comment:
Not sure about this event. Maybe `Shake()` should just return a `TransformSequence` to append possible completion callbacks to.

Reply:
Would it be better to just have the callback as a parameter in `Shake()` rather than returning the `TransformSequence`? Returning the sequence would allow for the caller to add their own transforms to the chain which might be more visibility than intended.

Reply:
I disagree, `Shake()` is just like any transform method and should return a `TransformSequence` for appending other transforms to it. (it should even be an extension following the transform patterns in `TransformableExtensions` and `TransformSequenceExtensions` rather than a method, but that can happen separately)

Reply:
Makes sense. I was looking at it like it only belonged in `ShakeContainer`. Considering it _should_ be part of `TransformableExtensions`/`TransformSequenceExtensions`, it makes way more sense to return the `TransformSequence`.

Reply:
Actually, now that the placeholder text response is no longer being used, the event is no longer in use, so I'll remove the event & return `Shake()` to it's original form. The change to return the `TransformSequence` from `Shake()` doesn't make sense here as it doesn't apply anymore, but while we're here I can open an issue/discussion about moving `Shake()` to somewhere in `TransformableExtensions`/`TransformSequenceExtensions` if you'd like.